src/tests/test_data/test_metadata_file_gcms_rerun.csv

```diff
@@ -0,0 +1,2 @@
+material_processing_type,raw_data_file,processed_data_file

```

Comment:
Please remove the material_processing column.  In a re-run scenario we won't touch those parts of the records (those _will_ deal with things before the MassSpectrometry records).